src/dns/server.rs

```diff
@@ -1401,13 +1446,13 @@ mod tests {
         });
     }
 
-    // fn new_large_response() -> Vec<Record> {
-    //     let mut out = Vec::new();
-    //     for i in 0..64 {
-    //         out.push(a(n("aaaaaaaaaaaa.aaaaaa."), ipv4(format!("240.0.0.{i}"))));
-    //     }
-    //     out
-    // }
+    fn new_large_response() -> Vec<IpAddr> {
+        let mut out = Vec::new();
+        for i in 0..244 {
```

Comment:
what is the significance of 244?

Reply:
Just needs to be a large number. I made it 256 to make it a bit less random (now its the max we can do in the right octet)